sidecars/java/Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN apk --no-cache add openjdk11 --repository=http://dl-cdn.alpinelinux.org/alpi
 ENV JAVA_HOME /usr/lib/jvm/default-jvm/
 ADD etc/before-start.sh /before-start.sh
 
+RUN wget -O ${HOME}/lombok.jar https://projectlombok.org/downloads/lombok-1.18.18.jar
```

Comment:
Let's make this an `ENV` variable at the top of the file so it's easily seen. Something like `LOMBOK_VERSION` or something, here's an example: https://github.com/eclipse/che-plugin-registry/blob/master/sidecars/kubernetes-tooling/Dockerfile#L13

Reply:
I updated the PR